package.json

```diff
@@ -0,0 +1,105 @@
+        "l10n",
+        "globalization",
+        "g11n",
+        "date",
+        "time",
+        "format",
+        "locale",
+        "translation",
+        "case"
+    ],
+    "homepage": "https://github.com/iLib-js/ilib-casemapper",
+    "bugs": "https://github.com/iLib-js/ilib-casemapper/issues",
+    "email": "marketing@translationcircle.com",
+    "license": "Apache-2.0",
+    "author": {
+        "name": "Edwin Hoogerbeets",
+        "web": "http://www.translationcircle.com/",
+        "email": "edwin@translationcircle.com"
+    },
+    "contributors": [
+        {
+            "name": "Edwin Hoogerbeets",
+            "email": "ehoogerbeets@gmail.com"
+        },
+        {
+            "name": "Goun Lee",
+            "email": "goun.lee@lge.com"
+        }
+    ],
+    "files": [
+        "src",
+        "lib",
+        "docs",
+        "README.md",
+        "LICENSE"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git@github.com:iLib-js/ilib-casemapper.git"
+    },
+    "scripts": {
+        "build": "npm run build:prod",
+        "build:prod": "grunt babel --mode=prod",
+        "build:dev": "grunt babel --mode=dev",
+        "build:test": "npm-run-all build:dev build:pkg build:webtest",
+        "build:pkg": "echo '{\"type\": \"commonjs\"}' > lib/package.json",
+        "build:webtest": "webpack-cli --env dev --config webpack-test.config.js",
+        "dist": "npm-run-all doc build:prod build:pkg; npm pack",
+        "test:cli": "LANG=en_US.UTF8 npm-run-all build:dev build:pkg ; bash test/testSuite.sh",
+        "test:web": "LANG=en_US.UTF8 npm run build:test ; open-cli ./test/testSuite.html ; open-cli ./test/testSuite.html -- firefox",
+        "test": "LANG=en_US.UTF8 npm-run-all build:dev test:cli",
+        "test:all": "LANG=en_US.UTF8 npm-run-all test:cli test:web",
+        "debug": "npm run build:dev ; node --inspect-brk test/testSuite.js",
+        "clean": "git clean -f -d src test ; rm -rf lib *.tgz",
+        "doc": "mkdir -p docs ; jsdoc2md -c jsdoc.json --separators --source src/* -m table > docs/CaseMapper.md",
+        "doc:html": "jsdoc -c jsdoc.json"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.22.11",
+        "@babel/preset-env": "^7.22.14",
+        "@babel/register": "^7.22.5",
+        "@babel/runtime": "^7.22.11",
+        "assertextras": "^1.1.0",
+        "babel-loader": "^9.1.3",
+        "babel-plugin-add-module-exports": "^1.0.4",
+        "babel-plugin-module-resolver": "^5.0.0",
+        "docdash": "^2.0.2",
+        "grunt": "^1.6.1",
+        "grunt-babel": "^8.0.0",
+        "grunt-cli": "^1.4.3",
+        "grunt-contrib-clean": "^2.0.1",
+        "grunt-contrib-jshint": "^3.2.0",
+        "grunt-contrib-nodeunit": "^5.0.0",
+        "grunt-contrib-uglify": "^5.2.2",
+        "jsdoc": "^4.0.2",
+        "jsdoc-babel": "^0.5.0",
+        "jsdoc-to-markdown": "^8.0.0",
+        "load-grunt-tasks": "^5.1.0",
+        "nodeunit": "^0.11.3",
```

Comment:
Wouldn't we rather go for Jest right away?

Reply:
This package is meant to run in node or in a browser as part of a webpacked app. As such, we have to test in a browser. So far, I haven't spent the time figuring out how to run jest in a browser. I know there are packages that allow you to do that, but I haven't figured that out yet. The packages I have converted to jest so far are the ones that only run on node in a command-line tool (loctool and its plugins, and soon i18nlint and its plugins).